swagger.json

```diff
@@ -5153,6 +5385,22 @@
             "in": "query",
             "type": "integer",
             "format": "int64"
+        },
+        "safetyScoreStartMsParam": {
+            "name": "startMs",
+            "description": "Timestamp in milliseconds representing the start of the period to fetch, inclusive. Used in combination with endMs.",
```

Comment:
Is there a maximum duration that this endpoint enforces?

Reply:
there isn't right now, we aren't too worried about expensive queries since we are now precomputing safety score. I will add it in later if we do enforce a max though